packages/nextjs/app/builders/0x5ba397662e0Dc4569199De084C340Cd15b151716/page.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+// import Image from "next/image";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const BuilderProfile: NextPage = () => {
```

Comment:
As there are more profile pages, you may want to consider giving it a more descriptive name.

Reply:
Understood, will use my Github username.